### ISSUE TITLE
SHLP demos are gone ??

### ISSUE THREAD

**Issue by choper725** [abap2UI5/abap2UI5#915]
wanted to do tests on the SHLP class, and tried to find the repo on app_tools,
seems to be missing ?

any way getting errors once runing class Z2UI5_CL_TOOL_APP_09,

wanted to test this along with all the major changes done...

**Comment by oblomov-dev** [comment]
yes its now here:
[a2UI5_demo-tools](https://github.com/oblomov-dev/a2UI5_demo-tools)
i try to split it more up to make it smaller, that its easier to maintain.
Thats why i have the table functions now here:
https://github.com/oblomov-dev/a2UI5-db_table_loader
i will check this later.

**Comment by oblomov-dev** [comment]
I cleaned up the tools repository now, but also check these two new repos dependent on what you need:
https://github.com/oblomov-dev/a2UI5-db_table_loader
https://github.com/oblomov-dev/a2UI5-mime_faker

**Comment by choper725** [comment]
nice ill take a look on those.

**Comment by choper725** [comment]
regarding the issue thx for updating demo_tools. works fine

**Comment by axelmohnen** [comment]
Hi @oblomov-dev,

this re-organization is a bit confusing.
All ABAP2UI5 related devs are in the "abap2UI5" account (like framework and demo apps).
And now the "ABAP2UI5" tools are under the "oblomov-dev" account?
Thanks,
Kind regards
Axel

**Comment by oblomov-dev** [comment]
yes understand, i am reorganize everything a little at the moment, also have in mind to make the tools repository smaller to make it easier to maintain. give me a few days and then we can check it again.

**Comment by oblomov-dev** [comment]
@axelmohnen created now a separated repository out of the search help. I don't want to have multiple functionalities in a single repository (makes it to difficult to maintain), thats why there is the split to table_loader, generic_search_help etc. 

I am the author of this repository now, but i am not sure if i am entitled to do this, because you made all the work ;) maybe its better when i transfer the repository to you? let me know if you want to have it.

https://github.com/oblomov-dev/a2UI5-generic_search_help

<img width="1198" alt="image" src="https://github.com/abap2UI5/abap2UI5/assets/102328295/68a609f0-4a8b-4c5e-8c96-495af460acf6">

**Comment by axelmohnen** [comment]
Hi @oblomov-dev 

I would prefer to have the repo in the ABAP2UI5 account.
In this way I can see all 2UI5 related repos in one single place.
Example:
_"https://github.com/abap2UI5/a2UI5-generic_search_help"_

![image](https://github.com/abap2UI5/abap2UI5/assets/22371169/d87682c8-9cf0-4fbc-906d-db032a83ee8d)

Thanks,
Axel

**Comment by oblomov-dev** [comment]
yes my idea is to put into the abap2UI5 account only the main functionality like the main repo, lauchpad extension, samples etc. but no applications. i have no time to maintain applications besides abap2UI5 too, and i only want to have actively maintained repositories in the abap2UI5 account.

maybe creating an account abap2UI5-addons/apps could be an option, but not everyone will put their app into this account and i also think it is the best when everyone keeps its apps in their accounts that everyone can see who is responsible and takes care of the repository.

but i think having a single place where all abap2UI5 applictions are listed is a nice idea and very helpful, but i am not sure if an github organization is the right place for this. i created a list [here](https://github.com/abap2UI5/abap2UI5-documentation/blob/main/links.md) and there is a list in abap2UI5 itself now:
<img width="779" alt="image" src="https://github.com/abap2UI5/abap2UI5/assets/102328295/412e2c6c-0388-4f1f-a3e2-c7d3567c95fa">

furthermore [this one](https://dotabap.org/) exists for abap open source in general, i am not a fan of having too many lists and places, so yes i am still not sure whats the best way to handle this and try to figure this out at the moment...

**Comment by axelmohnen** [comment]
Hi @oblomov-dev,
yes your right. Keep the core clean ;-)
In that case I will take over the "DDIC Searchhelp Generator" into my account and will create a PR for the link file.
Is this the right approach?
Thanks,
Axel

**Comment by oblomov-dev** [comment]
yes keep the core clean ;) this sounds like the right approach for now!

i think you need to delete your fork first before i can transfer:
<img width="600" alt="image" src="https://github.com/abap2UI5/abap2UI5/assets/102328295/8db5f62c-f830-47b8-888e-852518d3aa32">

**Comment by axelmohnen** [comment]
Hi @oblomov-dev 
My repo has been deleted. Could you please retry the transferring.
Thanks,
Axel

**Comment by axelmohnen** [comment]
OK, next try. I delete the fork "axelmohnen/abap2UI5-tools" as well...

**Comment by oblomov-dev** [comment]
ok looks better now:
<img width="921" alt="image" src="https://github.com/abap2UI5/abap2UI5/assets/102328295/b442e4f2-65bd-4a6f-a5ff-5977179f6d70">

**Comment by oblomov-dev** [comment]
Nice! if you want you can then send a PR to the link list:
https://github.com/abap2UI5/abap2UI5-documentation/blob/main/links.md
And to the build-in app list:
https://github.com/abap2UI5/abap2UI5/blob/main/src/01/03/z2ui5_cl_core_app_search.clas.locals_imp.abap

I think then it should be easy to find for everyone who want to use this functionality.

**Comment by oblomov-dev** [comment]
thank you!
Both PRs are merged, i think we can close this issue for now. just reopen it if you have further questions or ideas.
